.github/workflows/upload.yml

```diff
@@ -27,7 +27,9 @@ jobs:
         uses: actions/cache@v3
         with:
           path: downloads
-          key: plugin-downloads
+          key: ${{ runner.os }}-plugin-downloads-${{ hashFiles('**/plugin-releases.json') }}
```

Comment:
Curious why any of this needs to be cached, isn't it readily available via a `gh` call to https://github.com/bufbuild/plugins/releases ?